### PR TITLE
Re-run `flutter create` on examples to fill in icons and other assets

### PR DESCRIPTION
**Pull request by chinmaygarde**

### CODE REVIEW

File: examples/material_gallery/android/res/.empty [hunk 1]
```diff
@@ -0,0 +1 @@
+Place Android resources here (http://developer.android.com/guide/topics/resources/overview.html).
```
Comment (eseidelGoogle) [inline examples/material_gallery/android/res/.empty]: This is no longer necessary, or?

**Comment by eseidelGoogle** [comment]
I don't think the .empty files are needed.  Otherwise lgtm.

**Comment by chinmaygarde** [comment]
Ok. Rebased.

File: examples/fitness/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+# fitness
```
Comment (eseidelGoogle) [inline examples/fitness/README.md]: Might as well right a realer README. :)  But OK.

Fitness is a demo which shows text input and saving/reading from a json file.  Honestly we could probably delete the fitness example at this point.  @Hixie

**Comment by Hixie** [comment]
I'm not super happy about us checking in binaries into the flutter repo (though I guess it's too late now). It means every subsequent person who downloads our SDK has to download all these binaries.

cc @abarth.

@eseidelGoogle: Re fitness, I'm fine with dropping it, but cc @collinjackson and @vlidholt who I believe worked on it.

**Comment by eseidelGoogle** [comment]
@Hixie These binaries are part of the template project in flutter_tools. Thankfully git is smart enough to unique all of them :)

**Comment by collinjackson** [comment]
I think are two fitness apps, the one Viktor recently created in material_gallery and old one I created in examples. I think the one in examples isn't being maintained and can be dropped.

**Comment by eseidelGoogle** [comment]
Correct.  We have a Fitness Vignette in the Gallery app and a Fitness app in examples.  We wanted examples/fitness to be an every-day carry app, but never really finished it.  Lets just delete it.  Thanks.

**Comment by eseidelGoogle** [comment]
Done https://github.com/flutter/flutter/pull/2185

**Pull request merged**